Lib/asyncio/tasks.py

```diff
@@ -126,6 +131,14 @@ def __del__(self):
     def _repr_info(self):
         return base_tasks._task_repr_info(self)
 
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, value):
+        self._name = str(value)
```

Comment:
Is forcing this to a `str` really important enough to need a property? The name is basically only ever used for printing anyway, which will do the conversion when necessary.

Reply:
As I mentioned above I don't think we need a setter. A `Task.get_name()` function is enough for now.

Reply:
> I can think of a few use cases for this, like displaying the status of a task as it progresses.

Hm, interesting. OK, that's at least one valid use-case :)

> I could convert this to java-style setters and getters if you want, despite how much I despise them (not to mention the fact that threading also switched away from this behavior). But your call.

It's not about liking Java-style getters or not, it's about consistency.  If asyncio's APIs were using `@property` it'd be equally happy about that, but now we better follow the overall style and use `get_name()` and `set_name()`.